modules/ROOT/partials/configuration/newdocument_content.adoc

```diff
@@ -0,0 +1,21 @@
+[[newdocument_content]]
+== `+newdocument_content+`
+
+This option enables you to set the content that the editor will contain if you use the New document menu item. This can
+be used together with xref:content-behavior-options.adoc#editable_root[editable_root] to make sure there is always a editable field within the non-editable root.
+
+*Type:* `+String+`
+
+*Default value:* `+''+`
+
+=== Example: using `+newdocument_content+`
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  newdocument_content: '<p contenteditable="true">Editable content</p>' 
+});
```

Comment:
```suggestion
This option enables you to set the content that the editor will contain if you use the New document menu item.

*Type:* `+String+`

*Default value:* `+''+`

=== Example: using `+newdocument_content+`

This example shows `+newdocument_content+` used together with the xref:content-behavior-options.adoc#editable_root[`+editable_root+` option] to ensure that there is always an editable element within the non-editable root.

[source,js]
----
tinymce.init({
  selector: 'textarea',  // change this value according to your HTML
  editable_root: false,
  newdocument_content: '<p contenteditable="true">Editable content</p>' 
});
```
Moved the use case next to the relevant example. There should also be a link connected to "New document menu item", to a relevant section of the docs. And perhaps this option should be included there as well?